File: auto-docs/executables/matlab/YIOrRd_heatmap.m
signin('TestBot', 'r1neazxo9w')

size = 50;
z = zeros(size, size);
for r = 1:size
    for c = 1:size
        z(r,c) = r+c
    end
end

data = {...
  struct(...
    'z', z, ...
    'scl', 'YIOrRd', ...
    'type', 'heatmap')...
};
layout = struct('title', 'YIOrRd');

response = plotly(data, struct('layout', layout, 'filename', 'YIOrRd-heatmap', 'fileopt', 'overwrite', 'auto_open', 'false'));
plot_url = response.url

File: auto-docs/executables/r/YIOrRd_heatmap.r
library(plotly)

p <- plotly(username='TestBot', key='r1neazxo9w')

size <- 50
z <- matrix(0, size, size)
for(r in 1:size) {
    for(c in 1:size) {
        z[r, c] <- r+c
    }
}

data <- list(
  list(
    z = z, 
    scl = "YIOrRd", 
    type = "heatmap"
  )
)
layout <- list(title = "YIOrRd")

response <- p$plotly(data, kwargs=list(layout=layout, filename="YIOrRd-heatmap", fileopt="overwrite", auto_open="FALSE"))
url <- response$url
filename <- response$filename